### DISCUSSION TITLE
Animated typing indicator in message area

### DISCUSSION THREAD

**Discussion by Vinnie881** [advanced-chat/advanced-chat-components#413] [General]
I have been going through the js and vue, and I see no way to add a animated typing indicator in the message area.  I see it shows at the top on the room profile header, and on the left side navigation that a user is typing, but is there any slot I can modify or way to add a animated indicator someone is typing like most other chat apps?

**Comment by antoine92190** [comment]
I'm not sure where you would like to add that, and how it would look like. Could you send a video or screenshot?
